services/frontend-service/pkg/handler/handle_test.go

```diff
@@ -952,6 +952,106 @@ func TestServer_Handle(t *testing.T) {
 	}
 }
 
+type mockRolloutClient struct {
+	api.RolloutServiceClient
+	request  *api.GetStatusRequest
+	response *api.GetStatusResponse
+}
+
+func (m *mockRolloutClient) GetStatus(_ context.Context, in *api.GetStatusRequest, _ ...grpc.CallOption) (*api.GetStatusResponse, error) {
+	m.request = in
+	return m.response, nil
+}
+
+func TestServer_Rollout(t *testing.T) {
+	tests := []struct {
+		name                  string
+		req                   *http.Request
+		KeyRing               openpgp.KeyRing
+		signature             string
+		AzureAuthEnabled      bool
+		statusResponse        *api.GetStatusResponse
+		expectedResp          *http.Response
+		expectedBody          string
+		expectedStatusRequest *api.GetStatusRequest
+	}{
+		{
+			name: "propagates the environment group",
+			req: &http.Request{
+				Method: http.MethodPost,
+				URL: &url.URL{
+					Path: "/environment-groups/development/rollout-status",
+				},
+				Header: http.Header{
+					"Content-Type": []string{"application/json"},
+				},
+				Body: io.NopCloser(strings.NewReader(`{}`)),
+			},
```

Comment:
nitpick:
I think it would be easier to read if instead of `req *http.Request`, you just define a `requestBody string`, and make it either empty or fill it with the team.

Reply:
At the moment yes, however this test should be extended soon to cover more cases and then it's needed.